src/CoreImage/CIImageInitializationOptions.cs

```diff
@@ -34,53 +34,20 @@
 
 namespace XamCore.CoreImage {
 
-	public class CIImageInitializationOptions : DictionaryContainer
+	public partial class CIImageInitializationOptions
```

Comment:
Changing the base class of a public class is a breaking change?

Reply:
Mmh it shouldn't be a breaking change I generate the strong dictionary here: https://github.com/xamarin/xamarin-macios/pull/3017/files#diff-14b0758943a42f6b156f42dc8a2f8116R1274

`CIImageInitializationOptions.g.cs` has `public partial class CIImageInitializationOptions : DictionaryContainer`

Reply:
@chamons https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md <- it's in our wiki ;)

Reply:
Jenkins bots also produce the API diff to verify

Reply:
@spouliot The ? was "this certainly looks like a break, what am I missing" not a question on if removing the base class is breaking. :)